woocommerce/woocommerce
# [CYS] Update the woo.com themes link to include the back url

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45377



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Go to `WooCommerce > Home` and click on `Start customizing`.
4. Check the 3 paid themes below go to the `woo.com` theme pages and that you can see the `Back to Store` link.

<img width="1898" alt="Screenshot 2024-03-18 at 12 42 21" src="https://github.com/woocommerce/woocommerce/assets/186112/7741b3fb-4d99-46aa-9686-462b15cbe3f5">

5. Click on the `Back to Store` and make sure you are redirected to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
[CYS] Update the woo.com themes link to include the back url
#### Comment 

</details>
